nixos/modules/virtualisation/containers.nix

```diff
@@ -28,6 +28,31 @@ in
       description = lib.mdDoc "Enable the OCI seccomp BPF hook";
     };
 
+    cdi = mkOption {
+      type = types.attrs;
+      default = { };
+      description = lib.mdDoc ''
+        This option configures the Container Device Interface (CDI) of the system. Every key of the attribute set will be mapped to a file in /etc/cdi, and is required to be provided in JSON format.
```

Comment:
"This option configures" might be a bit redundant for the man page. Maybe decimate to "A declarative CDI specification. Each key corresponds to... Cf. the [specification](...) for ..."